framework/src/main/java/org/tron/common/validator/TransactionValidator.java

```diff
@@ -0,0 +1,49 @@
+package org.tron.common.validator;
+
+import javax.annotation.PostConstruct;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.tron.core.capsule.TransactionCapsule;
+
+@Component
+@Slf4j(topic = "transactionValidator")
+public class TransactionValidator {
+
+  @Autowired
+  private Validator<TransactionCapsule> dupTransactionValidator;
+  @Autowired
+  private Validator<TransactionCapsule> bigTransactionValidator;
+  @Autowired
+  private Validator<TransactionCapsule> expiredTransactionValidator;
+  @Autowired
+  private Validator<TransactionCapsule> contractSizeValidator;
+  @Autowired
+  private Validator<TransactionCapsule> taposTransactionValidator;
+  @Autowired
+  private Validator<TransactionCapsule> signatureValidator;
+
+  @PostConstruct
+  private void prepare() {
+    dupTransactionValidator
+        .nextValidator(bigTransactionValidator)
+        .nextValidator(expiredTransactionValidator)
+        .nextValidator(contractSizeValidator)
+        .nextValidator(taposTransactionValidator)
+        .nextValidator(signatureValidator);
+  }
```

Comment:
The way you assembled the validator, the coupling is relatively high, can it be changed to a dynamic loading form to reduce the code coupling